packages/twenty-front/src/modules/page-layout/widgets/widget-card/components/WidgetCard.tsx

```diff
@@ -0,0 +1,163 @@
+import { isPageLayoutInEditModeComponentState } from '@/page-layout/states/isPageLayoutInEditModeComponentState';
+import { useRecoilComponentValue } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValue';
+import styled from '@emotion/styled';
+import { type ReactNode } from 'react';
+import { isDefined } from 'twenty-shared/utils';
+
+import { type WidgetCardContext } from '../types/WidgetCardContext';
+
+export type WidgetCardProps = {
+  children?: ReactNode;
+  widgetCardContext?: WidgetCardContext;
```

Comment:
Should it be possible to render a widget card without providing a context? I feel we would prefer to force developers to specify which type of widget card they want to render.